css-three-column-layout/styles.css

```diff
@@ -0,0 +1,61 @@
+body {
+  background-color: #1996da;
+}
+
+h2 {
+  font-family: 'Lato', sans-serif;
+  font-weight: 300;
+  color: white;
+  text-align: center;
+  font-size: 3.5em;
+}
+
+h3 {
+  font-family: 'Lato', sans-serif;
+  font-weight: 300;
+  color: white;
+  font-size: 1.5em;
+  margin-bottom: 10px;
+}
+
+p {
+  font-family: 'Lato', sans-serif;
+  color: lightgray;
+  font-size: 1em;
+  max-width: 50%;
+  margin: auto;
+}
+
+.text-align {
+  text-align: center;
+  margin: auto;
+}
+
+.image {
+  width: 60%;
+  display: block;
+  margin: auto;
+}
+
+.white-bg {
+  margin: auto;
+  background-color: white;
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+  justify-content: center;
+  display: flex;
+}
+
+.container {
+  max-width: 70vw;
```

Comment:
While this does work here, I would largely avoid using the vw, vh, vmin, and vmax properties for container widths. fixed sizes are recommended